src/ex1/PokemonClient.js

```diff
@@ -0,0 +1,44 @@
+class PokemonClient {
+    constructor() {
+        this.API_BASE = "https://pokeapi.co/api/v2";
+    }
+
+    async getPokemon(pokemonId) {
+        try {
+            const response = await fetch(`${this.API_BASE}/pokemon/${pokemonId}`);
+            const result = await response.json();
+            return result.name;
+        } catch (error) {
+            const errorMessage = `Pokemon with ID ${pokemonId} was not found`;
+            return errorMessage;
+        }
+    }
+
+    async getAllPokemons(pokemonsIds) {
+        const allPromises = [];
+        let pokemons = [];
+        try {
+            pokemonsIds.forEach((pokemonId) => {
+                allPromises.push(fetch(`${this.API_BASE}/pokemon/${pokemonId}`));
+            });
+
+            await Promise.all(allPromises).then((responses) => {
+                responses.some((response) => {
```

Comment:
checking for responses.some means that even if some pokemon exist in the list it will still fail for all of them. Could you bypass it?
<img width="541" alt="image" src="https://user-images.githubusercontent.com/36815567/170819327-f776d04b-31af-4bd8-a0dc-55940e102f32.png">